--- origsrc/threadweaver-5.9.0/examples/ThumbNailer/AverageLoadManager.cpp	2015-04-04 06:51:28.000000000 -0500
+++ src/threadweaver-5.9.0/examples/ThumbNailer/AverageLoadManager.cpp	2015-04-26 13:22:02.526141000 -0500
@@ -46,7 +46,7 @@ QPair<int, int> AverageLoadManager::work
 
 void AverageLoadManager::update()
 {
-#ifdef Q_OS_UNIX
+#if defined(Q_OS_UNIX) && !defined(Q_OS_CYGWIN)
     double averages[3];
     if (getloadavg(averages, 3) == -1) {
         return;
